pages/dashboard/markets.tsx

```diff
@@ -0,0 +1,12 @@
+import DashboardLayout from 'sections/dashboard/DashboardLayout';
+import MarketsPage from 'sections/dashboard/Markets';
+
+type MarketsProps = React.FC & { getLayout: (page: HTMLElement) => JSX.Element };
+
+const Markets: MarketsProps = () => {
+	return <MarketsPage />;
```

Comment:
We should add a title to these pages:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/10401554/184708872-70950d02-f259-472a-8b6b-afca80c333d2.png">